pkgs/games/anki/bin.nix

```diff
@@ -0,0 +1,67 @@
+{ fetchurl, stdenv, lib, buildFHSUserEnv, appimageTools, writeShellScript, anki, undmg }:
+
+let
+  pname = "anki-bin";
+  version = "2.1.36";
+
+  tarball = fetchurl {
+    url = "https://github.com/ankitects/anki/releases/download/${version}/anki-${version}-linux.tar.bz2";
+    sha256 = "01xcjnfs5pfh7v0nkffw2wpl19l6pj9k3kxrcawv3cm42asy0mfz";
+  };
+
+  unpacked = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = tarball;
+
+    installPhase = ''
+      runHook preInstall
+
+      xdg-mime () {
+        echo Stubbed!
+      }
+      export -f xdg-mime
+
+      PREFIX=$out bash install.sh
+
+      runHook postInstall
+    '';
+  };
+
+  dmg = fetchurl {
+    url = "https://github.com/ankitects/anki/releases/download/${version}/anki-${version}-mac.dmg";
+    sha256 = "1i6iidm5h8r9g801mvqxi2av03qdw3lr28056fv5ixnb5dq2wqim";
+  };
+
+  meta = with lib; {
+    inherit (anki.meta) license homepage description longDescription;
+    name = "anki-bin";
+    inherit version; # FIXME buildFHSUserEnv doesn't take a version argument
+    platforms = with platforms; linux ++ darwin;
```

Comment:
This only works on `x86_64-linux` and `x86_64-darwin` as the binary release contains binary code specific to these platforms - please reflect this in `meta.platforms` so Raspberry Pi lovers like me won't be confused :sparkling_heart:

Reply:
Good point, didn't even think about that. We've got aarch64-darwin coming up too.

How would I differentiate that though? `broken = !stdenv.isx86_64`?

Reply:
I think it would be better to just declare `meta.platforms = [ "x86_64-linux" "x86_64-darwin" ];` - platforms are strings.